appcues/src/main/java/com/appcues/debugger/DebuggerStatusManager.kt

```diff
@@ -328,4 +360,27 @@ internal class DebuggerStatusManager(
             }
         }
     }
+
+    suspend fun checkPushValidation() {
+        if (pushConfigured == true) return
+
+        pushErrorText = null
+        pushValidationToken = "push-token"
```

Comment:
is this or the null value in L376 or 381 needed? It seems like it is really validated when the receive happens on L54 AppcuesFirebaseMessenger? I'm sort of confused by the validation flow here.